### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by talha2302**

### CODE REVIEW

File: code/fetcher.py [hunk 1]
```diff
@@ -1 +1,46 @@
-# Should use Requests3 library
+import requests
+from bs4 import BeautifulSoup
+
+
+class Meta(object):
```
Comment (ahmedbilal) [inline code/fetcher.py]: When we print Meta class, it doesn't print out like the one in README.md

Comment (talha2302) [reply code/fetcher.py]: I have added the __repr__ method in Meta class

File: code/fetcher.py [hunk 2]
```diff
@@ -9,6 +9,11 @@ def __init__(self, url, title="", keywords=[], description=""):
         self.keywords = keywords
         self.description = description
 
+    def __repr__(self):
+        print(f"Meta(url='{self.url}', title='{self.title)}',
```
Comment (ahmedbilal) [inline code/fetcher.py]: There is an error here.

File: code/fetcher.py [hunk 3]
```diff
@@ -10,7 +10,7 @@ def __init__(self, url, title="", keywords=[], description=""):
         self.description = description
 
     def __repr__(self):
-        print(f"Meta(url='{self.url}', title='{self.title)}',
+        print(f"Meta(url='{self.url}', title='{self.title}',
```
Comment (ahmedbilal) [inline code/fetcher.py]: Please use triple quotes """ instead of double quotes  " because the format string is across multiple lines.

Comment (talha2302) [reply code/fetcher.py]: Done

**Comment by ahmedbilal** [comment]
This is perfect.

**Pull request closed without merging**